src/views/task-list-page/task-list-filters/task-list-filters.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import PageFilters from '@/components/page-filters/page-filters';
+import PageSection from '@/components/page-section/page-section';
+
+import taskListFiltersConfig from '../config/task-list-filters.config';
+import taskListPageQueryParamsConfig from '../config/task-list-page-query-params.config';
+
+import { styled } from './task-list-filters.styles';
+
+export default function TaskListFilters() {
```

Comment:
please add small test for making sure component renders without errors.
This can help with coverage too.

Reply:
If it's a unit test, we'll probably be mocking Page Filters out, is that okay?